modules/utxo-lib/test/integration_local_rpc/parse.ts

```diff
@@ -306,6 +306,9 @@ describe(`regtest fixtures`, function () {
       const isDefault = version === getDefaultTransactionVersion(network);
       describe(`${getNetworkName(network)} fixtures (version=${version}, isDefault=${isDefault})`, function () {
         scriptTypes.forEach((scriptType) => {
+          if (scriptType === 'p2trMusig2') {
+            return;
+          }
```

Comment:
why?

Reply:
Transaction Builder is used here. It needs to be removed from test suite first. Comment is added.